lib/DDG/Spice/TwitchStreams.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::TwitchStreams;
+use strict;
+use DDG::Spice;
+
+name "TwitchStreams";
+source "http://www.twitch.tv";
+description "Get Twitch Stream info";
+primary_example_queries "twitch streams dota 2";
+category "special";
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TwitchStreams/TwitchStreams.pm";
+attribution github => ["https://github.com/dwaligon", "Jason O'Donnell"];
+
+triggers startend => ('twitch streams',
+		              'streams twitch',
+		              'twitchtv streams');
```

Comment:
@moollaza Here's what I see when I use twitch.tv as a trigger:
![ss](https://cloud.githubusercontent.com/assets/2160810/6985152/d69f8774-d9f5-11e4-9143-ac0151b38dd7.PNG)
Console:
"http://api.tvmaze.com/singlesearch/shows?q=streams%20dota"
"404 Not Found"